geomesa-convert/geomesa-convert-text/src/main/scala/org/locationtech/geomesa/convert/text/DelimitedTextConverter.scala

```diff
@@ -80,44 +76,16 @@ class DelimitedTextConverter(format: CSVFormat,
                              val validating: Boolean)
   extends LinesToSimpleFeatureConverter {
 
-  var curString: String = null
-  val q = Queues.newArrayBlockingQueue[String](32)
-  // if the record to write is bigger than the buffer size of the PipedReader
-  // then the writer will block until the reader reads data off of the pipe.
-  // For this reason, we have to separate the reading and writing into two
-  // threads
-  val writer = new PipedWriter()
-  val reader = new PipedReader(writer, options.pipeSize)  // record size
```

Comment:
@anthonyccri was there a specific reason for the piped reader/writer, or is it ok to remove it?